test/stampede_SUITE.erl

```diff
@@ -117,7 +123,6 @@ do_connect(N, Transport, Port, Timeout) ->
 	{ok, _} = Transport:connect("localhost", Port, [{active, false}], Timeout),
 	do_connect(N - 1, Transport, Port, Timeout).
 
-do_log(Pid) when is_pid(Pid) ->
-	logger:info("~p~n", [erlang:process_info(Pid)]);
-do_log(Port) when is_port(Port) ->
```

Comment:
So stampede doesn't do ports? We want ports to be closed as well.

Reply:
The `havoc` suite doesn't do ports, either. While `havoc` _can_ do that, we didn't configure it to do that because it won't just kill ports belonging to the target applications/supervisors, but any (tcp) port in the system.

I'll see what I can do to stampede later.
https://github.com/juhlig/stampede/issues/2

Reply:
Thought it did. We want it to do this so that listening sockets being closed are also tested.

Reply:
Stampede can now target ports as well.